data_source_obmcs_core_instances_test.go

```diff
@@ -84,15 +84,15 @@ func (s *DatasourceCoreInstanceTestSuite) TestAccDatasourceCoreInstance_basic()
 			{
 				ImportState:       true,
 				ImportStateVerify: true,
-				Config:            s.Config,
-			},
-			{
 				Config: s.Config + s.TokenFn(`
 				data "oci_core_instances" "t" {
 					compartment_id = "${var.compartment_id}"
 					availability_domain = "${data.oci_identity_availability_domains.ADs.availability_domains.0.name}"
 					display_name = "{{.token}}"
-					limit = 1
+					filter {
```

Comment:
As discussed, would be good to see what happens with metadata/extended_metadata.

Reply:
Attempting to filter on maps or lists of structured objects returns no results. Maybe thats not ideal behavior but it's not a realistic use case until tags are integrated anyway.